js/packages/web/src/components/Banner/index.tsx

```diff
@@ -7,21 +9,130 @@ export const Banner = ({
   actionComponent,
   children,
 }: {
-  src: string;
+  src?: string;
   headingText: string;
-  subHeadingText: string;
+  subHeadingText?: string;
   actionComponent?: JSX.Element;
   children?: React.ReactNode;
 }) => {
+  // mock storefront, should receive as prop or provider instead
+  const storefront = {
+    meta: {
+      social: {
```

Comment:
Add social as a top level field at the same level as meta.

Need to update the storefront type that is the repo.

```
interface SocialInfo {
  discord: string;
  twitter: string;
  website: string;
}
interface Storefront {
  social: SocialInfo;
}
```

Reply:
Not to hijack the PR (too much), but how about adding an integration key as well? Like this:
```
export interface StoreSocialInfo {
  instagram?: string;
  discord?: string;
  twitter?: string;
  website?: string;
}

export interface StoreIntegrations {
  ga4: string;
}

export interface Storefront {
  pubkey: string;
  subdomain: string;
  meta: StorefrontMeta;
  theme: StorefrontTheme;
  social?: StoreSocialInfo;
  integrations?: StoreIntegrations;
}
```